iphone/Classes/LocaleModule.h

```diff
@@ -22,6 +22,10 @@ JSExportAs(getString,
            -(NSString *)getString
            : (NSString *)key withHint
            : (id)hint);
+JSExportAs(parseDecimal,
+           -(NSNumber *)parseDecimal
+           : (NSString *)text withLocaleId
+           : (id)localeId);
```

Comment:
So I'm a bit conflicted over the use of `id` for the second optional argument. The JSC API doesn't give us a straight-forward way to handle optional arguments. I just tested, and if you declare it as an `NSString*` (which it should be), but you don't provide the arg - it converts the `undefined` value to `"undefined"` as a String. Which is clearly an annoyance (and probably why I did the same above for `getString`'s `hint` arg).

So I guess we have to either declare as `id` like this so undefined becomes `nil` - or not even define the optional argument and instead pull it out of `[JSContext.currentArguments]`.